Lesson9/anna/Lesson9-aysel/styles.css

```diff
@@ -0,0 +1,214 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  display: grid;
+  margin: 100px auto;
+  grid-gap: 10px 10px;
+  grid-template-columns: 2fr 4fr 11fr 4fr 2fr;
+}
+img {
+  max-width: 100%;
+}
+
+.post > * {
```

Comment:
I think there's a lot easier solution to this. For all content that is not large images, add a class. Then, set max-width of this class to 680px. Just make sure that images are not inside the class so they can take more space.